network-api/networkapi/buyersguide/pagemodels/get_category_og_image_upload_path.py

```diff
@@ -0,0 +1,10 @@
+from networkapi.utility.images import get_image_upload_path
```

Comment:
note that this is not a page model, but a helper function, let's put it in utils.py

Reply:
yea totally makes sense!